website/src/components/blog/postTypes.tsx

```diff
@@ -18,6 +18,8 @@ export interface Post {
     frontmatter: {
         title: string
         description: string
+        seoTitle: string
+        seoDescription: string
```

Comment:
I would recommend documenting these fields here:
Also should mark these as optional with `?`.
```suggestion
        /** Controls the page's `<title>` for SEO and the browser tab label. Defaults to {@link title}. */
        seoTitle?: string
        /** Controls the page's `<meta name="description">` for SEO. Defaults to {@link description}. */
        seoDescription?: string
```